src/Mod/Path/PathScripts/PathEngraveBase.py

```diff
@@ -75,7 +75,7 @@ def buildpathocc(self, obj, wires, zValues, relZ=False):
             if hasattr(obj, 'StartVertex'):
                 offset = DraftGeomUtils.rebaseWire(offset, obj.StartVertex)
 
-            edges = copy.copy(PathOpTools.orientWire(offset).Edges)
+            edges = copy.copy(offset.Edges)
```

Comment:
This is not a valid fix. You only get away with this if the edges happen to be oriented correctly to begin with which is unfortunately not guaranteed. The rest of the code relies on a proper edge orientation. You either have to add support for Bezier curves into flipEdge or make the rest of the code deal with misoriented edges in order to fix this properly.